bb-pr

```diff
@@ -14,6 +14,7 @@ set -eo pipefail
 BITBUCKET_API_URL="https://api.bitbucket.org/2.0/repositories"
 GIT_REMOTE=$(git remote get-url origin | cut -f2 -d':') || true
 BITBUCKET_SLUG=${GIT_REMOTE%.git}
+GIT_BRANCH=$(git branch --show-current)
```

Comment:
Stack overflow suggests as per our convo; though of course now we assume that original and not "bitbucket" is the remote branch

```suggestion
GIT_REMOTE_BRANCH=$(git rev-parse --abbrev-ref --symbolic-full-name @{u})
GIT_BRANCH="${GIT_REMOTE_BRANCH#origin/}"
```